.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    env:
+      GO111MODULE: on
+      TARGET: amd64
+        
+    strategy:
+#       max-parallel: 2
+      matrix:
+        go-version: [1.12.x, 1.13.x]
+        os: [ubuntu-16.04, ubuntu-18.04, ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    
+    - name: Checkout project  
+      uses: actions/checkout@v2
+      
+    - name: Clone API project
+      run: go get github.com/sodafoundation/api
+      
+    - name: Prerequites packages on ubuntu-16.04 
```

Comment:
Are these pre requisites installed only for Ubuntu 16.04? There are no conditional checks, hence the question.